package-lock.json

```diff
@@ -7332,13 +2802,13 @@
             "integrity": "sha1-qdhPJj7I/obS4WzCz36pq8IA1ho=",
             "dev": true,
             "requires": {
-                "eve": "git://github.com/adobe-webplatform/eve.git#eef80ed"
+                "eve": "eve@git://github.com/adobe-webplatform/eve.git#eef80ed"
```

Comment:
**nitpick:** this will introduce a deploy issue as SSH may not be used by the deploy env. Since only the package-lock file has been updated, I'd suggest reverting the change to this file.

![image](https://user-images.githubusercontent.com/1500098/113292621-0b59a800-92f5-11eb-8b61-b086923f643f.png)

Reply:
fixed